### PR TITLE
Revert "Added in gitignore"

### PR DESCRIPTION
**Pull request by briangonzalez**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/briangonzalez/termrc/pull/17" target="_blank">ReviewNinja

File: .gitignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-.DS_store
```
Comment (briangonzalez) [inline .gitignore]: !fix this doesn't look right

Comment (briangonzalez) [inline .gitignore]: !resolved How does this look?

**Comment by briangonzalez** [comment]
!star

File: .gitignore (hunk 1 above)
Comment (briangonzalez) [inline .gitignore]: !fix this is still wrong

**Comment by briangonzalez** [comment]
!unstar

File: .gitignore (hunk 1 above)
Comment (briangonzalez) [inline .gitignore]: !fix ok, really fixed this time

Comment (briangonzalez) [inline .gitignore]: !resolved really fixed

**Comment by briangonzalez** [comment]
:+1:

**Pull request closed without merging**